website/content/gateway/api_reference/resources/credits/recipient_information.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "recipient_information"
+date: 2023-07-20T13:01:49+01:00
+anchor: "credit_recipient_information"
+weight: 181
+---
+##### Recipient information
+See the partner guideline for more details.
+
+{{% description_list %}}
+
+{{% description_term %}}recipient[first_name] {{% regex %}}[\x20-\x7E]{1,35} [ASCII printable characters](https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}The recipient's legal first name.
+
+{{% regex_optional %}}It must not contain special characters (?, @, #, $, &, \*, etc.), be all numeric, be fictious, be a nickname or be incomplete.{{% /regex_optional %}}
+{{% regex_optional %}}Required for Mastercard Payment of Winnings (PoW) and for Mastercard cross-border non-PoW, i.e. it is only optional for Visa and for Mastercard domestic non-PoW.{{% /regex_optional %}}
+{{% regex_optional %}}Required, mutually with `recipient[last_name]`{{% /regex_optional %}}
+{{% regex_optional %}}Mutually exclusive with `recipient[name]`.{{% /regex_optional %}}
+{{% /description_details %}}
+
+{{% description_term %}}recipient[middle_name] {{% regex %}}[\x20-\x7E]{1,35} [ASCII printable characters](https://en.wikipedia.org/wiki/ASCII#ASCII_printable_characters){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}The recipient's legal middle name in case of any.
+{{% regex_optional %}}It must not contain special characters (?, @, #, $, &, \*, etc.), be all numeric, be fictious, be a nickname or be incomplete.{{% /regex_optional %}}
+{{% regex_optional %}}If set requires `recipient[first_name]` and `recipient[last_name]` to be set.{{% /regex_optional %}}
```

Comment:
```suggestion
{{% regex_optional %}}If set, it requires `recipient[first_name]` and `recipient[last_name]` to be set.{{% /regex_optional %}}
```

:man_shrugging: 

Also in other places.

Reply:
Fixed by [`6518cd9` (#208)](https://github.com/clearhaus/gateway-api-docs/pull/208/commits/6518cd9e8e27ffc1dc1aade64af9be464e63de63)